src/contracts/core/AllocationManager.sol

```diff
@@ -738,4 +738,42 @@ contract AllocationManager is
 
         return strategies;
     }
+
+    /// @inheritdoc IAllocationManager
+    function getMinimumSlashableStake(
+        OperatorSet memory operatorSet,
+        address[] memory operators,
+        IStrategy[] memory strategies,
+        uint32 futureBlock
+    ) external view returns (uint256[][] memory slashableStake) {
+        slashableStake = new uint256[][](operators.length);
+        uint256[][] memory delegatedStake = delegation.getOperatorsShares(operators, strategies);
+
+        for (uint256 i = 0; i < operators.length; i++) {
+            address operator = operators[i];
+            slashableStake[i] = new uint256[](strategies.length);
+
+            for (uint256 j = 0; j < strategies.length; j++) {
+                IStrategy strategy = strategies[j];
+
+                // Fetch the max magnitude and allocation for the operator/strategy.
+                // Prevent division by 0 if needed. This mirrors the "FullySlashed" checks
+                // in the DelegationManager
+                uint64 maxMagnitude = _maxMagnitudeHistory[operator][strategy].latest();
+                if (maxMagnitude == 0) {
+                    continue;
+                }
+
+                Allocation memory alloc = getAllocation(operator, operatorSet, strategy);
+
+                // If the pending change takes effect before `futureBlock`, include it in `currentMagnitude`
+                if (alloc.effectBlock <= futureBlock) {
+                    alloc.currentMagnitude = _addInt128(alloc.currentMagnitude, alloc.pendingDiff);
```

Comment:
we should only do this if the increase is negative. otherwise this isn't a minimum. 

Since 
1. an operator may queue an allocation that takes effect at block `n` and increases slashable stake by 10%
2. an AVS calls `getMinimumSlashableStake` for `futureBlock=n` at block `n-10` and gets back the stake, factoring in the 10% increase
3. the AVS slashes the operator at `n-5`, before the increase, thinking that the increase is what's at stake

Reply:
[93aa2af](https://github.com/Layr-Labs/eigenlayer-contracts/pull/889/commits/93aa2afd48cc33f7c9a244640bc6b0f7d202fb4f)